janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -583,4 +583,33 @@
             "Can be used when `" + REQUEST_TRACKER_CLASS + "` is set to `RequestLogger`.",
         ConfigOption.Type.LOCAL,
         Boolean.class);
+
+    ConfigNamespace EXECUTOR_SERVICE = new ConfigNamespace(
+        CQL_NS,
+        "executor-service",
+        "Configuration options for CQL executor service which is used to process CQL queries.");
+
+    ConfigOption<Integer> EXECUTOR_SERVICE_CORE_POOL_SIZE = new ConfigOption<>(
+        EXECUTOR_SERVICE,
+        "core-pool-size",
+        "Core pool size for CQL executor service.",
+        ConfigOption.Type.LOCAL,
+        Integer.class,
+        10);
+
+    ConfigOption<Integer> EXECUTOR_SERVICE_MAX_POOL_SIZE = new ConfigOption<>(
+        EXECUTOR_SERVICE,
+        "max-pool-size",
+        "Maximum pool size for CQL executor service.",
+        ConfigOption.Type.LOCAL,
+        Integer.class,
+        100);
+
+    ConfigOption<Long> EXECUTOR_SERVICE_KEEP_ALIVE_TIME = new ConfigOption<>(
+        EXECUTOR_SERVICE,
+        "keep-alive-time",
+        "Keep alive time in milliseconds for CQL executor service.",
```

Comment:
(nitpick) I think we might want to either paste the explanation from ThreadPoolExecutor here:

```
when the number of threads is greater than the core, this is the maximum time that excess idle threads will wait for new tasks before terminating
```
or give a pointer to the ThreadPoolExecutor javadoc. Otherwise by simply looking at the documentation itself, one might not know what it is.